ceph-pr-clang-tidy/build/build

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -ex
+
+# Clone the Ceph repository
+git clone https://github.com/ceph/ceph.git "$WORKSPACE/ceph"
```

Comment:
can you do a shallow clone of only the relevant branch?
this would save considerable amount of time